app/Jobs/Curations/SyncPhenotypes.php

```diff
@@ -4,27 +4,55 @@
 
 use App\Phenotype;
 use App\Curation;
+use App\Events\Curation\CurationPhenotypesUpdated;
 use Illuminate\Bus\Queueable;
 use Illuminate\Contracts\Queue\ShouldQueue;
 use Illuminate\Foundation\Bus\Dispatchable;
 use Illuminate\Queue\InteractsWithQueue;
 use Illuminate\Queue\SerializesModels;
+use Illuminate\Validation\ValidationException;
 
 class SyncPhenotypes implements ShouldQueue
 {
     use Dispatchable, InteractsWithQueue, Queueable, SerializesModels;
 
-    public function __construct(private Curation $curation, private $phenotypeIds)
+    private array $errors = [];
+
+    public function __construct(private Curation $curation, private array $phenotypeIds)
     {
     }
 
     public function handle()
     {
-        $phenotypes = Phenotype::whereIn('id', $this->phenotypeIds ?? []);
-        if (!$phenotypes && $phenotypes->count() > 0) {
+        if (!$this->phenotypeIdsAreValid()) {
+            throw ValidationException::withMessages($this->errors);
+        }
+
+        $previousIds = $this->curation->phenotypes->pluck('id');
+
+        if ($previousIds->toArray() == $this->phenotypeIds) {
             return;
         }
 
-        $this->curation->phenotypes()->sync($phenotypes->pluck('id'));
+        $this->curation->phenotypes()->sync($this->phenotypeIds);
+
+        CurationPhenotypesUpdated::dispatch($this->curation->fresh(), $previousIds);
+    }
+
+    private function phenotypeIdsAreValid()
```

Comment:
Add validation that checks that phenotype ids exist.  This wasn't applicable when we were storing mim numbers, and was missed in previous updates.